components/automate-ui/src/app/page-components/create-user-modal/create-user-modal.component.ts

```diff
@@ -125,6 +125,13 @@ export class CreateUserModalComponent implements OnInit, OnDestroy {
     }
   }
 
```

Comment:
Please fix--this was my oversight:
Change this:
https://github.com/chef/automate/blob/2e5e0ec16d4bda3b3f8b5f1e601181bc71a4fa27/components/automate-ui/src/app/page-components/create-user-modal/create-user-modal.component.ts#L42
to this:
```
// Must stay in sync with error checks in create-user-modal.component.html
```